src/bun.js/bindings/JSPropertyIterator.zig

```diff
@@ -90,5 +91,27 @@ pub fn JSPropertyIterator(comptime options: JSPropertyIteratorOptions) type {
 
             return name;
         }
+
+        /// "code" is not always an own property, and we want to get it without risking exceptions.
+        pub fn getCodeProperty(this: *@This()) ?bun.String {
```

Comment:
Instead of calling separately, maybe add this to `.next()` in `if (i >= this.len) { ... }`?

Reply:
we only want to do this in the one case for error-handling